web/src/auth/LoginPage.js

```diff
@@ -515,7 +531,7 @@ class LoginPage extends React.Component {
                   id="input"
                   prefix={<UserOutlined className="site-form-item-icon" />}
                   placeholder={(this.state.loginMethod === "verificationCode") ? i18next.t("login:Email or phone") : i18next.t("login:username, Email or phone")}
-                  disabled={!application.enablePassword}
+                  disabled={!showForm}
```

Comment:
Isn't `!showForm` always false?

Reply:
no, if we turn off all 3 login method, it'll be true, and can auto signin

Reply:
@notdu am I understanding right?

![image](https://github.com/casdoor/casdoor/assets/3787410/85e7a16a-6449-498d-bac1-c9e3372d7559)

Reply:
yes, you are right, i think we do not need `disable` attribute. I'll remove it